bin/impl/base.sh

```diff
@@ -14,30 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ -z $WI_HOME ]; then
-  echo "WI_HOME=$WI_HOME must be set!"
-  exit 1
-fi
-
-mkdir -p $WI_HOME/logs
-
-export DATA_CONFIG=$WI_HOME/conf/data.yml
-if [ ! -f $DATA_CONFIG ]; then
-  echo "You must create $DATA_CONFIG"
-  exit 1
-fi
+: ${WI_HOME?"WI_HOME must be set"}
```

Comment:
I don't quite understand this, but I get the gist of it and I like it.  Its much more concise